rest-backend/src/main/java/com/sap/psr/vulas/backend/model/AffectedLibrary.java

```diff
@@ -82,6 +82,10 @@
 	@JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss.SSSZ", timezone="GMT")
 	private java.util.Calendar createdAt;
 	
+	@Temporal(TemporalType.TIMESTAMP)
+	@JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss.SSSZ", timezone="GMT")
+	private java.util.Calendar modifiedAt;
```

Comment:
In the future we should consistently use either `lastModified` or `modifiedAt`.

Reply:
I used modifiedAt in line with createdAt to denote the database level timestamps for creation and modification, resp. , of a single entity. The lastXXX, e.g., lastScan and lastVulnChange are timestamps  are instead the result of a computation .